t-1 name_task: markup, add styles, add assets, create widgets

maxexc opened the pull request:
markup, add styles, add assets, create widgets

maxexc committed:
t-1 name_task: markup, add styles, add assets, create widgets

StanislavSpasonov commented on lib/scr/features/main_page/main_page.dart:
```diff
@@ -1,13 +1,72 @@
+              icon: AppIcons.locationCity,
+              onPressed: () {
+                //todo add list city
+                print('onpress ');
```
remove print

StanislavSpasonov commented on linux/CMakeLists.txt:
```diff
@@ -0,0 +1,139 @@
+# Project-level configuration.
```
delete the linux folder with git add to git ignore

StanislavSpasonov commented on macos/.gitignore:
```diff
@@ -0,0 +1,7 @@
+# Flutter-related
```
delete the mac folder with git. add to git ignore

StanislavSpasonov commented on pubspec.yaml:
```diff
@@ -1,8 +1,8 @@
 name: weather_app_dart
-description: A new Flutter project.
+description: weather project.
 # The following line prevents the package from being accidentally published to
```
delete all comments

maxexc committed:
t_1 additional commit

corrected pubspec.yaml,
add exceptions in .gitignore,
removed 41 line in main_page.dart,
deleted the linux and macos folders
